bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss

```diff
@@ -0,0 +1,419 @@
+//////////////////////////
```

Comment:
You have seen it more often than it should be replicated. However in this case we are pretty much starting from scratch here. We'll need to add a SASS compiler to django-bulbs as I was just using the SASS binary locally to make edits.